package.json

```diff
@@ -1,41 +1,24 @@
 {
-  "name": "rel-run",
-  "version": "1.0.0",
-  "description": "",
-  "type": "module",
-  "main": "src/index.js",
+  "name": "rel",
+  "private": true,
+  "version": "0.0.0",
+  "description": "The modern backend framework for frontend developers.",
+  "main": "index.js",
+  "devDependencies": {
+    "@types/node": "^17.0.5",
+    "typescript": "^4.5.4"
+  },
+  "scripts": {
+    "build": "npm run build --workspaces"
+  },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/rel-js/rel-run.git"
+    "url": "git+https://github.com/rel-js/monorepo.git"
   },
   "author": "",
-  "license": "ISC",
+  "license": "MIT",
   "bugs": {
-    "url": "https://github.com/rel-js/rel-run/issues"
+    "url": "https://github.com/rel-js/monorepo/issues"
   },
-  "homepage": "https://github.com/rel-js/rel-run#readme",
-  "dependencies": {
-    "@genql/cli": "^2.8.0",
-    "@genql/runtime": "^2.8.0",
-    "@go-trace/tracer": "https://gitpkg.now.sh/rel-js/trace/tracer-npm-package?main",
-    "@graphback/codegen-schema": "https://gitpkg.now.sh/rel-js/graphback/packages/graphback-codegen-schema?master",
-    "@graphql-tools/schema": "^8.3.1",
-    "@nodelib/fs.walk": "^1.2.8",
-    "@ptkdev/logger": "^1.8.0",
-    "altair-fastify-plugin": "^4.2.1",
-    "fastify": "^3.25.0",
-    "fastify-cors": "^6.0.2",
-    "graphback": "https://gitpkg.now.sh/rel-js/graphback/packages/graphback",
-    "graphql": "^15.8.0",
-    "graphql-redis-subscriptions": "^2.4.2",
-    "graphql-yoga": "^2.0.0-alpha.2",
```

Comment:
while you are at it, can you update the graphql-yoga dependency to "2.0.0-alpha.5"?

Reply:
Good catch. Fixed.